scripts/logs-server.py

```diff
@@ -0,0 +1,33 @@
+# Python script to create a webserver which serves only 1 file over HTTP
+# (uncrypted), and returns a 404 error for all other requests.
+# Designed for serving uProxy cloud install logs to monitor install progress,
+# but can be re-used for other purposes.
+#
+# Usage: python logs-server.py <PORT> <FILE>
+#   e.g. python logs-server.py 8000 /directory/file
+
+from SimpleHTTPServer import SimpleHTTPRequestHandler
+import BaseHTTPServer
+import sys
+
+class SingleFileRequestHandler(SimpleHTTPRequestHandler):
+  restricted_file = ''
+  def end_headers(self):
+    # Allow HTTP requests to come from any origin.
+    self.send_header('Access-Control-Allow-Origin', '*')
+    SimpleHTTPRequestHandler.end_headers(self)
+  def do_GET(self):
```

Comment:
I see. It's just that this approach requires the client (your website) to know the path of the logfile - which might be a nuisance in future if that changes. Since it's just serving this file, you could probably get away with just using netcat, e.g.:
```
while true ; do nc -l 80 < test.html ; done
```
(from https://gist.github.com/willurd/5720255#gistcomment-1878879)